src/public/components/requests/classes.js

```diff
@@ -1,5 +1,14 @@
 import axios from 'axios';
 
+const getAttendanceRecordDate = async (queryData) => {
+  try {
+    const reponse = await axios.post('/getAttendanceRecordDate', queryData) 
```

Comment:
should be axios.get and using queries https://expressjs.com/en/api.html#req.query